Update: Add Toggle for Editor Orientation

jennsun opened the pull request:
https://app.clickup.com/t/xfhx48

Changed "Editor Form on Left" and "Editor Form on Right" separate radio buttons into a single toggle-switch for editor's orientation. Toggle-switch was a checkbox designed using CSS and correct orientation of editor was determined in onSubmit function by whether switch was toggled or not.

jennsun committed:
Update: Add Toggle for Editor Orientation

simanthastew commented on example_site/src/components/ConfigurationControls.css:
```diff
@@ -19,3 +19,94 @@ button.configuration-submit-button {
+input:checked + .slider .off
+{
+  display: none;
+}
```
This little red symbol means there should be an empty line at the end of the file!

![image](https://user-images.githubusercontent.com/22031072/121688622-aa3b1700-ca91-11eb-8538-7a86173a2e71.png)

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -46,7 +47,7 @@ export default function ConfigurationControls({ isOpen }) {
   } = watchFields;
 
-  const onSubmit = async (formData) => {
+   const onSubmit = async (formData) => {
```
Looks like you added an extra space in here - that's what the brighter green spot it 

![image](https://user-images.githubusercontent.com/22031072/121688515-8d064880-ca91-11eb-9a52-cb75ab5d1d0d.png)

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -78,10 +79,13 @@ export default function ConfigurationControls({ isOpen }) {
     setAccount(null);
 
+    /* Correct orientation for editor */
+    var finalOrientation = orientation ? "right" : "left";
```
We should be using `const` to declare this variable - let's talk about why during our morning meeting! But here are a few quick reasons:
1. This variable is not being reassigned 
2. We want the variable to be as tightly scoped as possible (although it won't matter much in this case)

This is probably a good time to talk about scoping. I think it would be good for you to do a little research today on:
1. The differences between `var`, `let`, and `const`
2. Function scope vs. block scope

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -78,10 +79,13 @@ export default function ConfigurationControls({ isOpen }) {
     setPartnerID(partnerID);
     setAccount(null);
 
+    /* Correct orientation for editor */
```
Very minor, but we use `//` to start in-line comments

simanthastew commented on example_site/src/components/ConfigurationControls.css:
```diff
@@ -19,3 +19,94 @@ button.configuration-submit-button {
+  transform: translate(40px, -35px);
+  top: 50%;
+  left: 50%;
+  font-size: 16px;
+  font-family: Verdana, sans-serif;
```
Do you need this font information?

simanthastew commented on example_site/src/components/ConfigurationControls.css:
```diff
@@ -19,3 +19,94 @@ button.configuration-submit-button {
+  display: none;
+}
+
+.on, .off
```
We would typically structure this like so:
```
.on,
.off {
    color: ....
}
```

simanthastew commented on example_site/src/components/ConfigurationControls.css:
```diff
@@ -19,3 +19,94 @@ button.configuration-submit-button {
+/* Alternate "Right" and "Left" text 
+depending on switch toggle */
+.on
+{
```
Let's make sure all of the opening brackets are on the same line as the class name!

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -166,27 +170,18 @@ export default function ConfigurationControls({ isOpen }) {
-        <label className="form-label" labelfor="leftOrientation">
-          <input
-            id="leftOrientation"
+        <h4>Form location: </h4>
```
Let's change this to `Editor orientation`

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -404,4 +399,4 @@ export default function ConfigurationControls({ isOpen }) {
     </form>
   );
-}
+}
```
Newline!

simanthastew commented on example_site/src/components/ConfigurationControls.js:
```diff
@@ -166,27 +170,18 @@ export default function ConfigurationControls({ isOpen }) {
+            type="checkbox"
+            ref={register}
+          /> 
+          <div class="slider round">
```
Let's talk about descriptive class names!

simanthastew reviewed (changes requested):
Nice job! Mostly some small stuff that we can go over together.

jennsun committed:
Update: PR Review
Fixed empty line/spacing issues, used const for finalOrientation since it is not reassigned, fixed commenting and structure formatting, removed font CSS, adjusted title to 'Editor Orientation', used more descripive class name for switch on/off classes

simanthastew commented on example_site/src/components/ConfigurationControls.css:
```diff
@@ -85,28 +85,23 @@ input:checked + .slider:before {
-.on, .off
-{
+.on, 
+.off {
```
Don't forget to update these!

simanthastew reviewed (changes requested):
Nice work - the only thing that's left is updating those class names
